### PR TITLE
Traceability agent docker image fix

### PR DESCRIPTION
**Pull request by rathnapandi**
Traceability agent docker image fix

### CODE REVIEW

File: build/webmethods_traceability_agent.yml [hunk 1]
```diff
@@ -8,7 +20,7 @@ webmethods_traceability_agent:
     deployment: ${CENTRAL_DEPLOYMENT:prod}
     environment: ${CENTRAL_ENVIRONMENT:""}
     environmentID: ${CENTRAL_ENVIRONMENTID} # only required when CENTRAL_USAGEREPORTING_OFFLINE=true
-    agentName: ${CENTRAL_AGENTNAME:""}
+    agentName: ${CENTRAL_AGENTNAME:"webmethods-agent-traceability"}
```
Comment (vivekschauhan) [inline build/webmethods_traceability_agent.yml:23]: This should be left empty and be overridden using the env var override. If not specified the SDK bootstrapping will create a agent resource using the environment name config

File: build/webmethods_traceability_agent.yml [hunk 2]
```diff
@@ -23,8 +35,8 @@ webmethods_traceability_agent:
       url: ${CENTRAL_AUTH_URL:https://login.axway.com/auth}
       realm: ${CENTRAL_AUTH_REALM:Broker}
       clientId: ${CENTRAL_AUTH_CLIENTID:""}
-      privateKey: ${CENTRAL_AUTH_PRIVATEKEY:/keys/private_key.pem}
-      publicKey: ${CENTRAL_AUTH_PUBLICKEY:/keys/public_key.pem}
+      privateKey: ${CENTRAL_AUTH_PRIVATEKEY:keys/private_key.pem}
```
Comment (vivekschauhan) [inline build/webmethods_traceability_agent.yml:38]: Should be /keys...

File: build/webmethods_traceability_agent.yml [hunk 3]
```diff
@@ -60,7 +63,7 @@ webmethods_traceability_agent:
     healthCheckInterval: ${STATUS_HEALTHCHECKINTERVAL:30s}
   log:
     level: ${LOG_LEVEL:info}
-    format: ${LOG_FORMAT:json}
+    format: ${LOG_FORMAT:line}
```
Comment (vivekschauhan) [inline build/webmethods_traceability_agent.yml:66]: Any reason, not to use json as default ?

Comment (rathnapandi) [reply build/webmethods_traceability_agent.yml:66]: Nothing specific, i added it for better readability.

File: build/webmethods_traceability_agent.yml (hunk 2 above)
Comment (rathnapandi) [reply build/webmethods_traceability_agent.yml:38]: fixed

File: build/webmethods_traceability_agent.yml (hunk 1 above)
Comment (rathnapandi) [reply build/webmethods_traceability_agent.yml:23]: fixed

**Review by vivekschauhan — APPROVED** [review]

**Pull request merged**